tests/moduleTests/stream.js

```diff
@@ -405,4 +405,16 @@ export default function tests() {
         const sf_offsets = Array.from(sf).map(n => n.offset);
         assert.deepEqual(sf_offsets, [0, 0, 2, 4, 4, 6, 8, 8]);
     });
+
+    test('music21.stream.Stream isGapless', assert => {
+        const testOne = new music21.stream.Stream();
+        const n1 = new music21.note.Note('C');
+        const n2 = new music21.note.Note('D');
+        testOne.insert(0, n1);
+        testOne.insert(0, n2);
+        assert.equal(testOne.isGapless(testOne), true);
+        const n3 = new music21.note.Note('E');
+        testOne.insert(10.0, n3);
+        assert.equal(testOne.isGapless(testOne), false);
+    });
```

Comment:
Would you mind also testing the return value of `findGaps()`, verifying that the stream as the correct number of elements, etc.?